core/lib/common/archiver.js

```diff
@@ -1,3 +1,23 @@
+/**
+ * # Send reports/artifacts from testbot back to client
+ *
+ * By default, serial logs (given that the hardware is set up correctly), and the
+ * logs from the tests will be sent back to the client that started the test, upon
+ * the test finishing. Other artifacts can be sent back to the client using the
+ * `archiver` method. This method is available within any test:
+ *
+ * @example
+ * ```js
+ * this.archiver.add(`path/of/FILE`)
+ * ```
+ *
+ * Using this method, at the end of the test, any artifacts added to the archive are compressed and
+ * downloaded by the client. These are available in the `workspace/reports` directory at the end of
+ * the test. This can also be helpful when storing diagigggddddd
```

Comment:
I think there's a typo here ;D

Reply:
I thought I didn't know a cat for the excuse the cat ran over my keyboard. 
Nonetheless, resolved.